ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
Agree, this also makes our current xtest builds fail (just tried this patch on QEMU).